pod/perlfunc.pod

```diff
@@ -3896,10 +3896,11 @@ X<lc> X<lowercase>
 
 =for Pod::Functions return lower-case version of a string
 
-Returns a lowercased version of EXPR.  This is the internal function
-implementing the C<\L> escape in double-quoted strings.
+Returns a lowercased version of EXPR.  If EXPR is omitted, uses
+L<C<$_>|perlvar/$_>.
 
-If EXPR is omitted, uses L<C<$_>|perlvar/$_>.
+    my $str = lc("Perl is GREAT"); # "perl is great"
+    my $str = lc(undef);           # undef
```

Comment:
This should not be included as an example, since this is a string function and `undef` is not a string. This will warn and return an empty string.